server/text_generation_server/models/flash_causal_lm.py

```diff
@@ -910,9 +911,9 @@ def generate_token(
                 # Copy batch.input_ids to prefill_token_indices
                 if prefill_logprobs:
                     if len(batch) > 1:
-                        prefill_tokens_indices[
-                            out_start_index : out_end_index - 1
-                        ] = batch.input_ids[start_index + 1 : start_index + out_length]
+                        prefill_tokens_indices[out_start_index : out_end_index - 1] = (
```

Comment:
okay we're going to need to force lint everything with some standard instead of using each other's editor's default :)

Reply:
ahh I totally agree we should standardize our formatters. Currently I'm using Black out of the box. Is there an existing config I should use?

Reply:
Black is fine, we just need to enforce it repo wide in the CI. (We need to pin a revision though, black is not great at backward compatibility)